README.md

```diff
@@ -737,6 +738,73 @@ Requires
 * [Asterisk](http://www.asterisk.org/) with configured AMI interface (manager.conf)
 * pyst2 -  powerful Python abstraction of the various Asterisk APIs (pip install pyst2)
 
+## `boxcar`
+
+The `boxcar` service will use the boxcar notification service to send messages.  It was one of the first IOS notification services..
+
+The paylod (message) of the mqtt topic, can use the "|" seperator to contain a notification title, a long message and a url.  
```

Comment:
Is it worth continuting with this PR given that Boxcar is: A not multiplatform (IOS only) and B, has been pulled from the IOS app store.

I didn't realise this before making the changes.